systems/robotInterfaces/@Biped/Biped.m

```diff
@@ -108,7 +108,10 @@
            0 0 0 0 0 -1;
            0 0 0 0 0 1;
            0, 1/(params.max_step_width-params.nom_step_width), 0, 0, 0, 1/params.max_outward_angle;
-           0, 1/(params.min_step_width-params.nom_step_width), 0, 0, 0, 1/params.max_outward_angle];
+           0, 1/(params.min_step_width-params.nom_step_width), 0, 0, 0, 1/params.max_outward_angle;
+           params.max_outward_angle / (params.max_forward_step), 0, 0, 0, 0, 1;
+           -params.max_outward_angle / (params.max_forward_step), 0, 0, 0, 0, 1;
```

Comment:
Dividing through by max_outward_angle would be more consistent with the existing constraints, but without knowing the variable order it looks potentially reasonable.